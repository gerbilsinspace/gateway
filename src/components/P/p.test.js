import React from 'react';
import { shallow } from 'enzyme';
import P from './index';

it('renders without crashing', () => {
  shallow(<P />);
});