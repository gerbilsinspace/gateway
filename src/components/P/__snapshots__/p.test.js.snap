// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`P should match snapshot with no data 1`] = `
<p
  className="sc-bdVaJa bDWFJH"
/>
`;
